bundles/net.tourbook/src/net/tourbook/importdata/DialogEasyImportConfig.java

```diff
@@ -4810,7 +4717,7 @@ private void update_UI_From_Model_IL() {
                   linkTourType.setData(DATA_KEY_SPEED_TOUR_TYPE_INDEX, speedTTIndex);
                   spinnerAvgSpeed.setData(DATA_KEY_SPEED_TOUR_TYPE_INDEX, speedTTIndex);
                   comboCadence.setData(DATA_KEY_SPEED_TOUR_TYPE_INDEX, speedTTIndex);
-                  _actionTTSpeed_Delete[speedTTIndex].setData(DATA_KEY_SPEED_TOUR_TYPE_INDEX, speedTTIndex);
+                  _actionTTSpeed_Delete[speedTTIndex].setData(speedTTIndex);
```

Comment:
Because the [setData ](https://github.com/mytourbook/mytourbook/blob/4a4bf14e8fbe9c72b553117abf3427354f7859eb/bundles/net.tourbook/src/net/tourbook/importdata/DialogEasyImportConfig.java#L424)function doesn't use this parameter `key`

Reply:
I didn't saw that this is a private function, because calling `setData()` for a widget could be dangerous as it could overwrite data, therefore I normally use a key

Reply:
so you want me to revert it ?